### DISCUSSION TITLE
Spool remaining percentage

### DISCUSSION THREAD

**Discussion by jjvelar** [greghesp/ha-bambulab#671] [Q&A]
Hi,
Is it possible to get remaining percentage per spool?
Thanks and best regards,

José

**Comment by AdrianGarside** [comment]
Only if you are using Bambu spools. It's on the extended attributes for the tray.

**Reply by jjvelar** [reply]
Thanks for answering.
I have a couple of Bambu spools loaded in my AMS Lite but I can't find it. Only a boolean for "empty" is available.

![image](https://github.com/user-attachments/assets/dcc61dff-02ff-46d6-bf5a-56c4ff23462c)

**Reply by AdrianGarside** [reply]
Ah. A1. I recently removed it for the AMS life as I was told it doesn’t have the ability to measure remaining and the value was always fixed at 100.

**Comment by jjvelar** [comment]
Would you like me to test it for you?

**Reply by AdrianGarside** [reply]
Do Bambu Studio or Handy show remaining

**Comment by jjvelar** [comment]
no. but it shows even less information than the current list of attributes
per tray you are currently providing.
it only shows filament name, color, min/max nozzle temperature and serial
number

**Comment by AdrianGarside** [comment]
Then that's in line with what another A1 owner told me - that the AMS Lite doesn't track filament remaining.
